packages/cli/snippets/custom-auth-otp.js

```diff
@@ -0,0 +1,31 @@
+const authentication = {
+  type: 'custom',
+  test: {
+    url: 'https://{{bundle.authData.subdomain}}.example.com/api/accounts/me.json',
+  },
+  // The function to call out to verify the OTP code.
+  customConfig: {
+    sendCode: {
+      method: 'POST',
+      url: 'https://{{bundle.inputData.subdomain}}.example.com/api/otp/verify',
```

Comment:
The API endpoint for `sendCode` isn't actually to "verify". So the "verify" in the API URL can be misleading. The endpoint is to send the one-time code to the user. Also should we add `headers` and `body` too?

```suggestion
      url: 'https://{{bundle.inputData.subdomain}}.example.com/api/otp/send',
      headers: {
        Authorization: `Bearer {{process.env.API_KEY}}`
      },
      body: {
        to_phone_number: '{{bundle.inputData.phone_number}}',
        code: '{{bundle.inputData.code}}',
      },
```